src/tike/operators/convolution.py

```diff
@@ -6,11 +6,44 @@
 
 
 class Convolution(Operator):
-    """2D Convolution operator with linear interpolation."""
+    """A 2D Convolution operator with linear interpolation.
+
+    Compute the product two arrays at specific relative positions.
+
+    Attributes
+    ----------
+    nscan : int
+        The number of scan positions at each angular view.
+    fly : int
+        The number of consecutive scan positions that describe a fly scan.
+    nmode : int
+        The number of probe modes per scan position.
+    probe_shape : int
+        The pixel width and height of the (square) probe illumination.
+    nz, n : int
+        The pixel width and height of the reconstructed grid.
```

Comment:
Say number of pixels for clarity? To avoid understanding of a width of 2 cm, 100 mm, etc. for example.

Reply:
Which of the following:
1. "The width and height in pixels of the (square) probe illumination." 
2. "The width and height in number of pixels of the (square) probe illumination."
3. "The number of pixels in the width of the (square) probe illumination."